df.announcements.xml

```diff
@@ -1,12 +1,24 @@
 <data-definition>
     <enum-type type-name='announcement_type' base-type='int16_t'>
+        <enum-attr name='alert_type' type-name='announcement_alert_type' default-value='GENERAL'/>
+
         <enum-item name='NONE' value='-1'/>
         <enum-item name='REACHED_PEAK'/>
```

Comment:
`REACHED_PEAK` (0) corresponds to `GENERAL` (0) by falling through the switch and hitting the `return 0` at the bottom

Reply:
Okay. This nearly doubles the lines of the enum definition, though.